package.json

```diff
@@ -22,6 +22,7 @@
     "@nestjs/common": "9.0.0",
     "@nestjs/config": "2.3.3",
     "@nestjs/core": "9.0.5",
+    "@nestjs/cqrs": "10.0.1",
```

Comment:
I think you should install `@nestjs/cqrs` with version corresponding to `nest/core` version. In this case `9.0.x`